### PR TITLE
black_hole:Delete an unnecessary check that will cause panic.

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve? 
Delete an unnecessary check that will cause panic.

### What is changed and how it works?
Delete the unnecessary check.

### Check List 

Tests 

 - No code

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]

File: cdc/sink/black_hole.go [hunk 1]
```diff
@@ -37,15 +37,6 @@ type blackHoleSink struct {
 }
 
 func (b *blackHoleSink) EmitRowChangedEvents(ctx context.Context, rows ...*model.RowChangedEvent) error {
-	checkpointTs := atomic.LoadUint64(&b.checkpointTs)
```
Comment (ben1009) [inline cdc/sink/black_hole.go:40]: do we need add a test like `assert.noPanic`

**Review by ben1009 — APPROVED** [review]

**Review by liuzix — APPROVED** [review]

**Comment by amyangfei** [comment]
```
[2021-06-11T10:01:56.297Z] ----------------------------------------------------------------------
[2021-06-11T10:01:56.297Z] FAIL: pool_test.go:115: workerPoolSuite.TestMultiError
[2021-06-11T10:01:56.297Z] 
[2021-06-11T10:01:56.297Z] pool_test.go:137:
[2021-06-11T10:01:56.297Z]     c.Assert(err, check.ErrorMatches, ".*ErrWorkerPoolHandleCancelled.*")
[2021-06-11T10:01:56.297Z] ... error string = "context canceled"
[2021-06-11T10:01:56.297Z] ... regex string = ".*ErrWorkerPoolHandleCancelled.*"
[2021-06-11T10:01:56.297Z] 
[2021-06-11T10:01:56.297Z] OOPS: 15 passed, 1 FAILED
[2021-06-11T10:01:56.297Z] --- FAIL: TestSuite (21.77s)
[2021-06-11T10:01:56.297Z] FAIL
[2021-06-11T10:01:56.297Z] FAIL	github.com/pingcap/ticdc/pkg/workerpool	21.816s
[2021-06-11T10:01:56.297Z] FAIL
```

**Pull request merged**